pex.go

```diff
@@ -144,8 +145,11 @@ func (me *pexMsgFactory) PexMsg() *pp.PexMsg {
 // Per-torrent PEX state
 type pexState struct {
 	sync.RWMutex
-	tail *pexEvent     // event feed list
-	hold []pexEvent    // delayed drops
+	tail *pexEvent  // event feed list
```

Comment:
This one here, btw, is not related to throttling: it's needed to send out the follow-up deltas every minute, as provisioned in the BEP. It's good to recover the logic if it got lost; however there should be code elsewhere which uses this field and which are not being added in this PR.